tests/test_core.py

```diff
@@ -7299,7 +7299,7 @@ def test_async(self, emterpretify=False):
 
     self.do_run(src, 'HelloWorld!99')
 
-    if emterpretify:
+    if self.is_wasm_backend() or emterpretify:
```

Comment:
We can now run these tests on the WASM backend to ensure `ccall` use works correctly.